helm/cluster-vsphere/files/etc/ssh/sshd_config

```diff
@@ -1,3 +1,4 @@
+# DEPRECATED - remove once CP and workers are rendered with cluster chart
```

Comment:
I've added these deprecation markers to make it easier to find things to be deleted at a later date